client/src/Plumeria.jsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+
+export default function Plumeria() {
+  const [plumerias, setPlumerias] = useState();
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+
+  useEffect(() => {
+    async function loadPlumeria() {
+      try {
+        const response = await fetch('/api/:categoryId/product');
```

Comment:
You want to send an actual categoryId, not `:categoryId`. You'll likely figure that out how to do that once you get further along in your implementation.